resources/istio-kyma-patch/README.md

```diff
@@ -0,0 +1,14 @@
+# Istio Kyma patch
+
+## Overview
+
+This chart packs [patch script](../../components/istio-kyma-patch/README.md) as kubernetes job.
+
+## Patches
+
+By default following patches are applied:
+ * CRD `policies.authentication.istio.io` is required. This means that security in istio must be enabled.
```

Comment:
``The `policies.authentication.istio.io` CRD is required. This means that security in Istio must be enabled.``